cristin-project/src/main/java/no/unit/nva/cristin/projects/model/nva/NvaContributor.java

```diff
@@ -51,12 +68,82 @@ public boolean equals(Object o) {
         }
         NvaContributor that = (NvaContributor) o;
         return getType().equals(that.getType())
-                && getIdentity().equals(that.getIdentity())
-                && getAffiliation().equals(that.getAffiliation());
+               && getIdentity().equals(that.getIdentity())
+               && getAffiliation().equals(that.getAffiliation());
     }
 
-    @Override
-    public int hashCode() {
-        return Objects.hash(getType(), getIdentity(), getAffiliation());
+    public CristinPerson toCristinPerson() {
```

Comment:
Identity is a Person object. Maybe we can create a Person.toCristinPerson? We can extract roles separate as that is not part of identity